Gemfile

```diff
@@ -28,6 +28,14 @@ group :development do
   gem "puppet-module-posix-dev-r#{minor_version}", '~> 0.4',     require: false, platforms: [:ruby]
   gem "puppet-module-win-default-r#{minor_version}", '~> 0.4',   require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-module-win-dev-r#{minor_version}", '~> 0.4',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
+  gem "puppet-lint-trailing_newline-check", '~> 1.1.0',          require: false
```

Comment:
I don't see a .sync.yml along with this?
```
Gemfile:
  optional:
    ':development':
      - gem: 'puppet-lint-trailing_newline-check'
```
Would be the preference

Reply:
Then you can just run `pdk update` and it'll go and generate all of those gemfile changes